source/plugins/base/metadata.yml

```diff
@@ -21,10 +21,10 @@ inputs:
       * `header`, which usually contains username, two-week commits calendars and a few additional data
       * `activity`, which contains recent activity (commits, pull requests, issues, etc.)
       * `community`, which contains community stats (following, sponsors, organizations, etc.)
-      * `repositories`, which contains repositories stats (license, forks, stars, etc.)
-      * `metadata`, which contains informations about generated metrics
+      * `repositories`, which contains repository stats (license, forks, stars, etc.)
```

Comment:
Alternatively this could be `repositories' stats`, to mean "stats of multiple repositories"

Reply:
I feel `repository stats` sounds better.

Reply:
I actually agree, just making sure the alternatives are out there for the maintainer to review.

Reply:
:+1: